student.rb

```diff
@@ -0,0 +1,10 @@
+class Student < Person
+  def initialize(classroom, name, age)
+    super(name, age)
```

Comment:
- [ ] I think a student will require to obtain permission from the parents if he/she is not of age before renting a particular book. Hence you'll need to pass `parent_permission` as a parameter in the `super()` method to inherit from the parent class(Person) 
- **Hints**

```ruby
def initialize(classroom, name = "Unknown", age, parent_permission = true)  
    super(age, name, parent_permission)
    @classroom = classroom
end
```